lib/actions/ssh.coffee

```diff
@@ -53,6 +54,11 @@ module.exports =
 			boolean: true
 			description: "don't use the proxy configuration for this connection.
 				Only makes sense if you've configured proxy globally."
+	,
+			signature: 'command'
+			parameter: 'command'
+			description: 'command to run after connecting to the device (only in the host OS)'
```

Comment:
How come it has to be in the host OS?

Reply:
@CameronDiver connecting to the container with the current "exec" method does not seem to run the command passed. Will have to check with @wrboyce whether that's really the case, but right now it doesn't seem to work. I hope later we can expand on this.

Reply:
There should be a method, but as you say Will is the one to ask.

Reply:
Checking with Will, but there doesn't seem to be a way currently as we are connecting inside the container.

Reply:
Nope, it won't work with the container.